extensions/dependencies/logic_proof.html

```diff
@@ -1,3 +1,4 @@
+<!-- TODO(gvishal): Add extension resource prefix here -->
```

Comment:
I meant add cache slugs to it. Wasn't able to figure out which controller this goes into